code/game/gamemodes/dynamic/dynamic_unfavorable_situation.dm

```diff
@@ -66,6 +66,7 @@
 		/datum/round_event_control/immovable_rod,
 		/datum/round_event_control/meteor_wave,
 		/datum/round_event_control/portal_storm_syndicate,
+		/datum/round_event_control/operative, //ORBSTATION ADDITION
```

Comment:
If this is a heavy ruleset you shouldn't need to add it here because it already has the ability to call any valid heavy ruleset

Reply:
Lone op isn't actually a ruleset at all, just an event. Thus it won't actually ever be called unless it's here.